services/frontend-service/pkg/cmd/server.go

```diff
@@ -127,8 +131,39 @@ type Auth struct {
 	HttpServer http.Handler
 }
 
+func getActionAuthor(r *http.Request) *auth.User {
+	ctx := r.Context()
+
+	var c Config
+
+	err := envconfig.Process("kuberpult", &c)
```

Comment:
does this read the env variables multiple times? as in line https://github.com/freiheit-com/kuberpult/blob/c7ccf6ba104b638ff358d351b90dfd7e06567a8c/services/frontend-service/pkg/cmd/server.go#L44